extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -115,17 +116,47 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
       (event) => event.stopPropagation(),
       true
     );
+
+    const DURATION_MS = 1500;
+    let startTime;
+    const animateFrame = (currTime) => {
+      // Set current startTime if not defined.
+      if (!startTime) {
+        startTime = currTime;
+      }
+      const elapsed = currTime - startTime;
+      if (elapsed < DURATION_MS) {
+        // Value between 0 and 1;
+        const timePercentage = elapsed / DURATION_MS;
+        // example with Math.sin animation
+        // Converts oscillation between 1 and -1, to 0 and 30.
+        const val = map(Math.cos(timePercentage * Math.PI * 2), 1, -1, 0, 30);
+        this.inputEl_.value = val;
+        this.onDrag_();
```

Comment:
Don't think we should call `this.onDrag` since we're not dragging the slider. Using the `--fraction` variable directly could work, and we avoid the logic for the bubble and the `if` statement